pkg/search/proxy/store/util.go

```diff
@@ -244,8 +305,8 @@ func RecoverClusterResourceVersion(obj runtime.Object, cluster string) bool {
 		return false
 	}
 
-	m := make(map[string]string)
-	err = json.Unmarshal(decoded, &m)
+	var m map[string]string
+	err = unmarshalRvs(decoded, &m)
```

Comment:
why do you think `contine` also shall change to this format?

Reply:
whatis contine???

Reply:
My mistake. Forget it.